app.py

```diff
@@ -21,6 +21,12 @@
 PATH_JWT_SECRET_KEY = PATH_DIR / 'auth_jwt_secret_key'
 PATH_PASSWORD = PATH_DIR / 'auth_password'
 
+# check if the files exists
+if not PATH_PASSWORD.exists():
+    raise FileNotFoundError(f'Password file not found, Please readme on how to set up the password file')
```

Comment:
Just a comment on grammar, similar for the next string too

```suggestion
    raise FileNotFoundError(f'Password file not found, please check the README.md on how to set up the password file')
```